packages/@aws-cdk/aws-ses/README.md

```diff
@@ -1,2 +1,123 @@
 ## The CDK Construct Library for AWS Simple Email Service
 This module is part of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) project.
+
+### Email receiving
+Create a receipt rule set with rules and actions:
+```ts
+const bucket = new s3.Bucket(this, 'Bucket');
```

Comment:
Could you please move this into a file named:

```
test/example.receiving.lit.ts
```

And add the requirement imports and `Stack` declaration around it?

You can mark the piece of code that is relevant for the example with:

```
/// !show
...code goes here
/// !hide
```

And at this location in the README add:

```
[example of setting up a receive rule](test/example.receiving.lit.ts)
```

-----

This will make sure that the example actually compiles, and if we make breaking API changes, the compiler failure will force us to update the example code.